integration_test/login_test.dart

```diff
@@ -45,9 +54,41 @@ void main() {
         of: find.byKey(const Key('irma_dialog')),
         matching: find.byType(YiviThemedButton),
       ));
+
+      // Press "Forgot pin"
+      final forgotPinLinkFinder = find.text('I forgot my PIN').hitTestable();
+      expect(forgotPinLinkFinder, findsOneWidget);
+      await tester.tapAndSettle(forgotPinLinkFinder);
+
+      // Expect to be on the reset pin screen
+      final forgotPinScreenFinder = find.byType(ResetPinScreen);
+      expect(forgotPinScreenFinder, findsOneWidget);
+
+      // Expect content the right content
```

Comment:
You could name it something like the following to make it more explanatory:
`// Expect forgotten PIN explanation content`